### ISSUE TITLE
Gremlin extensions not loaded when gremlin.sh is run from directory other than dist root

### ISSUE THREAD

**Issue by rjbriody** [thinkaurelius/titan#922]
Observed in titan-console-0.9.0-M1.

Extensions are loaded when gremlin.sh is started from the dist root:

```
./bin/gremlin.sh 

         \,,,/
         (o o)
-----oOOo-(3)-oOOo-----
plugin activated: tinkerpop.server
plugin activated: tinkerpop.utilities
INFO  com.tinkerpop.gremlin.hadoop.structure.HadoopGraph  - HADOOP_GREMLIN_LIBS is set to: /home/bob/aurelius/titan/dists/titan-console-0.9.0-M1/bin/../ext/hadoop-gremlin:/home/bob/aurelius/titan/dists/titan-console-0.9.0-M1/bin/../ext/titan-all
plugin activated: tinkerpop.hadoop
plugin activated: aurelius.titan
plugin activated: tinkerpop.tinkergraph
```

...but when gremlin is started from anywhere else the extensions are not loaded

```
./titan-console-0.9.0-M1/bin/gremlin.sh 

         \,,,/
         (o o)
-----oOOo-(3)-oOOo-----

```

I have verified that the $DIR and $CLASSPATH vars in gemlin.sh look correct.

**Comment by dalaro** [comment]
Opened TP3 PR: 

https://github.com/tinkerpop/tinkerpop3/pull/513

**Comment by dalaro** [comment]
tinkerpop/tinkerpop3#513 was merged.

I copied the relevant `gremlin.sh` tweak into Titan's copy of the script via 8a778950afb0bf2b9f603b7f158acafefffb3975.  Once we update to a TP3 version that has tinkerpop/tinkerpop3#513, no further changes in Titan besides that commit should be necessary to get the benefit.

**Comment by dalaro** [comment]
Reopening this -- should keep it unresolved until we get to a TP version that includes the upstream PR.

**Comment by dalaro** [comment]
This fix survived into the TP3 3.0.0.M8 release (checked in gremlin-console-3.0.0.M8-distribution.zip).

Titan's copy of the fix is also still OK (checked in titan09 @ cff4c8588e7a1e6eead650af45d2af5399a7ad5e).

This is resolved.
